specification/validated-relationships-service-api.yaml

```diff
@@ -199,6 +230,236 @@ paths:
                 $ref: '#/components/schemas/OperationOutcome'
```

Comment:
Comment not related to this line

We have expression defined for the operational outcome but none of our errors currently return it, so maybe it should be removed
Operational Outcome confluence page -> https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?pageId=791874870
Operational Outcome object -> https://github.com/NHSDigital/validated-relationships-service-api/pull/78/files#diff-e143931886bb4215c74e499a54357c5e498c103ff719c91bc8e7a977b3768517R827